src/iceberg/request-upgrader.service.js

```diff
@@ -102,10 +102,21 @@ export default class IcebergRequestUpgrader {
       options: cloneDeep(actionOptions),
     };
 
-    merge(action.options, {
-      headers: IcebergRequestUpgrader.buildHeaders(apiOptions, cleanCache),
-      serviceType: 'apiv6',
-    });
+    const defaultOptions = {
+      isArray: false,
+      transformResponse: (
+        data,
+        headers,
+        status,
+      ) => (status === 200 ? { data: JSON.parse(data), headers: headers() } : data),
```

Comment:
Need to confirm if statusCode 200 is a requirement or not.

Reply:
> Need to confirm if statusCode 200 is a requirement or not.

@antleblanc No need for status checking in fact, updating the PR